content/en/blog/falco-okta-identity/index.md

```diff
@@ -0,0 +1,168 @@
+
+
+![](https://github.com/falcosecurity/falco-website/tree/master/content/en//blog/falco-okta-identity/images/okta10.png)
+
+
+Finally, the entire process of manually executing Okta search queries in the web UI, or managing intricate detection scripts, can be time-consuming and often results in coverage gaps. Falco offers a solution by delivering a nearly real-time detection engine that enables the use of macros and lists for complex querying. For instance, consider the task of verifying whether the user 'Nigel Douglas' is logging in from their usual IP address. Instead of navigating through complex Okta queries, you can simply use the actor ID, cross-referencing it with the typical IP they use for sign-ins, and also taking into account the context of their access, such as interactions with the 'Admin Dashboard' or other elements within the Okta user interface. 
+
+This is how Okta queries can be structured:
+
+```
+eventType eq "user.session.access_admin_app" and client.ip_address eq "78.xx.xxx.249" and target.id eq "00u9xcz5aphuQ8ZQq5d7" and outcome.result eq "SUCCESS"
+```
+
+Yet, if our aim is to identify successful login attempts from questionable geographic regions or IP addresses, we must integrate lists and diligently keep them up-to-date. 
+This is where the following Falco rule proves its worth:
+
+
+```
+- rule: Suspicious Login for Nigel Douglas
+  desc: Detect suspicious login attempts from known suspicious IPs
+  condition: okta.evt.type = "user.session.access_admin_app" and okta.client.ip in (suspicious_ips) and okta.target.user.id= "00u9xcz5aphuQ8ZQq5d7"
+  output: Suspicious IP Inbound Request
+(okta.actor.name=%okta.actor.name, okta.client.ip=%okta.client.ip, okta.target.user.id=%okta.target.user.id, okta.target.user.name=%okta.target.user.name, okta.app=%okta.app, okta.evt.type=%okta.evt.type)
+  priority: CRITICAL
+  tags: [custom_rule, mitre_initial_access, T1078, valid_accounts]
+  source: okta
+```
+
+Next, we create a list that compiles all potentially identified malicious actors who may attempt an account takeover on a legitimate account, which should typically be accessed from a consistent IP address. This list logic can be applied to geolocations, such as countries, instead of specific IPs. In both scenarios, the list is named 'suspicious_ips' and is referred to in the Falco rule conditions as follows:
```

Comment:
```suggestion
Next, we create a list that compiles all potentially identified malicious actors who may attempt an account takeover on a legitimate account, which should typically be accessed from a consistent IP address. This list logic can be applied to geolocations, such as countries, instead of specific IPs. In both scenarios, the list is named `suspicious_ips` and is referred to in the Falco rule conditions as follows:
```